src/mats_l1_processing/pointing.py

```diff
@@ -23,9 +16,6 @@ def add_channel_quaternion(CCDitem):
     return
 
 
-
-
-
 def pix_deg(ccditem, xpixel, ypixel):
```

Comment:
pix_deg is only used in the MATS-utility-functions repository and was defined in  /MATS-utility-functions/src/mats_utils/geolocation/coordinates.py originally. 
Should we move it back to this file to avoid cross imports between repositories ?

Reply:
It seems to only be referenced in the test. I'll remove it.